relayer/naive-strategy.go

```diff
@@ -13,7 +14,7 @@ import (
 )
 
 // UnrelayedSequences returns the unrelayed sequence numbers between two chains
-func UnrelayedSequences(ctx context.Context, src, dst *Chain, srcChannel *chantypes.IdentifiedChannel) (*RelaySequences, error) {
+func UnrelayedSequences(ctx context.Context, src, dst *Chain, srcChannel *chantypes.IdentifiedChannel) *RelaySequences {
```

Comment:
We never return a nil value, so this can have a return signature of `RelaySequences` instead of `*RelaySequences`.